Make target `format` silently fails in GitHub actions

<img width="2002" height="1138" alt="Image" src="https://github.com/user-attachments/assets/cd1a96c7-b83d-479d-a7ac-87c63c87c813" />

Relevant logs from GitHub [actions build](https://github.com/DataBiosphere/azul/actions/runs/21928304603/job/63326132699):
```
python scripts/generate_environment_boot.py
2026-02-12 00:15:34,729    INFO MainThread __main__: No change for 'azul_python_version', still set to '3.13.11'
2026-02-12 00:15:34,729    INFO MainThread azul.docker: Resolving 'python' image TagImageRef(registry='docker.io', username='library', repository=('python',), tag='3.13.11-slim-trixie') …
2026-02-12 00:15:34,729    INFO MainThread azul.docker: Resolved 'python' image to DigestImageRef(registry='122796619775.dkr.ecr.us-east-1.amazonaws.com', username='docker.io', repository=('library', 'python'), digest='sha256:a4c611a15c9c46597c5b519f559d6634143c2f065d3c7419192b9fdc9cef87af')
2026-02-12 00:15:34,729    INFO MainThread __main__: No change for 'azul_docker_version', still set to '29.1.5'
2026-02-12 00:15:34,729    INFO MainThread __main__: No change for 'azul_terraform_version', still set to '1.13.5'
docker run \
    --attach stdout \
    --rm \
    --volume $(python scripts/resolve_container_path.py /home/runner/work/azul/azul):/home/developer/azul \
    --workdir /home/developer/azul \
    $(AZUL_DEBUG=0 python -m azul 'docker.resolve_docker_image_for_launch("pycharm")') \
    /opt/pycharm/bin/format.sh -r -settings .pycharm.style.xml -mask '*.py' src scripts test lambdas/layer/app.py lambdas/indexer/app.py lambdas/service/app.py .flake8/azul_flake8.py environment.py deployments/anvilbox/environment.py deployments/anvildev.browser/environment.py deployments/anvildev.gitlab/environment.py deployments/anvildev.shared/environment.py deployments/anvildev/environment.py deployments/anvilprod.browser/environment.py deployments/anvilprod.gitlab/environment.py deployments/anvilprod.shared/environment.py deployments/anvilprod/environment.py deployments/dev.browser/environment.py deployments/dev.gitlab/environment.py deployments/dev.shared/environment.py deployments/dev/environment.py deployments/hammerbox/environment.py deployments/prod.browser/environment.py deployments/prod.gitlab/environment.py deployments/prod.shared/environment.py deployments/prod/environment.py deployments/sandbox/environment.py deployments/tempdev.browser/environment.py deployments/tempdev.gitlab/environment.py deployments/tempdev.shared/environment.py deployments/tempdev/environment.py .github/workflows/schedule.py terraform/data_sources.tf.json.template.py terraform/sqs.tf.json.template.py terraform/common.tf.json.template.py terraform/authentication.tf.json.template.py terraform/s3.tf.json.template.py terraform/elasticsearch.tf.json.template.py terraform/providers.tf.json.template.py terraform/step_function.tf.json.template.py terraform/api_gateway.tf.json.template.py terraform/cloudwatch.tf.json.template.py terraform/lambda_layer.tf.json.template.py terraform/backend.tf.json.template.py terraform/dynamo.tf.json.template.py terraform/gitlab/gitlab.tf.json.template.py terraform/shared/shared.tf.json.template.py terraform/browser/browser.tf.json.template.py lambdas/indexer/.chalice/config.json.template.py lambdas/service/.chalice/config.json.template.py .github/pull_request_template.md.template.py resources/public_sources.json.template.py resources/environ.json.template.py
2026-02-12 00:15:35,294    INFO MainThread __main__: Trying to extract ID of current container from /proc/self/mountinfo
Traceback (most recent call last):
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 74, in <module>
    main(sys.argv[1])
    ~~~~^^^^^^^^^^^^^
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 68, in main
    host_path = resolve_container_path(container_path)
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 56, in resolve_container_path
    container_id = one(container_ids)
  File "/home/runner/work/azul/azul/.venv/lib/python3.13/site-packages/more_itertools/more.py", line 627, in one
Error:     raise too_short or ValueError('too few items in iterable (expected 1)')
ValueError: too few items in iterable (expected 1)
2026-02-12 00:15:35,506    INFO MainThread azul.docker: Resolving 'pycharm' image TagImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), tag='2025.2.6-72') …
2026-02-12 00:15:35,506    INFO MainThread azul.docker: Resolved 'pycharm' image to DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:10d283bdd592e87eb3b8820d5a81aa5670d56bcefd5af3129f822dd5aedb38a9')
2026-02-12 00:15:35,506    INFO MainThread azul.docker: Pulling image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:10d283bdd592e87eb3b8820d5a81aa5670d56bcefd5af3129f822dd5aedb38a9') …
2026-02-12 00:16:04,374    INFO MainThread azul.docker: Pulled image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:10d283bdd592e87eb3b8820d5a81aa5670d56bcefd5af3129f822dd5aedb38a9')
docker: invalid spec: :/home/developer/azul: empty section between colons

Run 'docker run --help' for more information
make: *** [Makefile:238: format] Error 125
make: Entering directory '/home/runner/work/azul/azul/lambdas'
for d in indexer service; do make -C $d openapi || ! break; done
make[1]: Entering directory '/home/runner/work/azul/azul/lambdas/indexer'
python /home/runner/work/azul/azul/scripts/generate_openapi_document.py

```

**EDIT: @achave11-ucsc**

See [comment-below](https://github.com/DataBiosphere/azul/issues/7776#issuecomment-3898695910) for occurrence details.

The first occurrence in `develop` was after the changes, accurately referenced in the #7777 description (6caccad62), were merged to develop in commit ef9068d. The [GitHub actions test build](https://github.com/DataBiosphere/azul/actions/runs/21233401098/job/61096226738#step:5:655) for ef9068d silently failed (and started failing since):

```
2026-01-22 02:11:43,863    INFO MainThread __main__: No change for 'azul_docker_version', still set to '28.5.2'
2026-01-22 02:11:43,863    INFO MainThread __main__: No change for 'azul_terraform_version', still set to '1.13.5'
docker run \
    --attach stdout \
    --rm \
    --volume $(python scripts/resolve_container_path.py /home/runner/work/azul/azul):/home/developer/azul \
    --workdir /home/developer/azul \
    $(AZUL_DEBUG=0 python -m azul 'docker.resolve_docker_image_for_launch("pycharm")') \
    /opt/pycharm/bin/format.sh -r -settings .pycharm.style.xml -mask '*.py' src scripts test lambdas/layer/app.py lambdas/indexer/app.py lambdas/service/app.py .flake8/azul_flake8.py environment.py deployments/anvilbox/environment.py deployments/anvildev.browser/environment.py deployments/anvildev.gitlab/environment.py deployments/anvildev.shared/environment.py deployments/anvildev/environment.py deployments/anvilprod.browser/environment.py deployments/anvilprod.gitlab/environment.py deployments/anvilprod.shared/environment.py deployments/anvilprod/environment.py deployments/dev.browser/environment.py deployments/dev.gitlab/environment.py deployments/dev.shared/environment.py deployments/dev/environment.py deployments/hammerbox/environment.py deployments/prod.browser/environment.py deployments/prod.gitlab/environment.py deployments/prod.shared/environment.py deployments/prod/environment.py deployments/sandbox/environment.py deployments/tempdev.browser/environment.py deployments/tempdev.gitlab/environment.py deployments/tempdev.shared/environment.py deployments/tempdev/environment.py .github/workflows/schedule.py terraform/lambda_layer.tf.json.template.py terraform/cloudwatch.tf.json.template.py terraform/data_sources.tf.json.template.py terraform/backend.tf.json.template.py terraform/sqs.tf.json.template.py terraform/elasticsearch.tf.json.template.py terraform/step_function.tf.json.template.py terraform/common.tf.json.template.py terraform/dynamo.tf.json.template.py terraform/api_gateway.tf.json.template.py terraform/authentication.tf.json.template.py terraform/s3.tf.json.template.py terraform/providers.tf.json.template.py terraform/gitlab/gitlab.tf.json.template.py terraform/shared/shared.tf.json.template.py terraform/browser/browser.tf.json.template.py lambdas/indexer/.chalice/config.json.template.py lambdas/service/.chalice/config.json.template.py .github/pull_request_template.md.template.py
2026-01-22 02:11:44,399    INFO MainThread __main__: Trying to extract ID of current container from /proc/self/mountinfo
Traceback (most recent call last):
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 74, in <module>
    main(sys.argv[1])
    ~~~~^^^^^^^^^^^^^
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 68, in main
    host_path = resolve_container_path(container_path)
  File "/home/runner/work/azul/azul/scripts/resolve_container_path.py", line 56, in resolve_container_path
    container_id = one(container_ids)
  File "/home/runner/work/azul/azul/.venv/lib/python3.13/site-packages/more_itertools/more.py", line 627, in one
Error:     raise too_short or ValueError('too few items in iterable (expected 1)')
ValueError: too few items in iterable (expected 1)
2026-01-22 02:11:44,594    INFO MainThread azul.docker: Resolving 'pycharm' image TagImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), tag='2025.2.6-71') …
2026-01-22 02:11:44,594    INFO MainThread azul.docker: Resolved 'pycharm' image to DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467')
2026-01-22 02:11:44,594    INFO MainThread azul.docker: Pulling image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467') …
2026-01-22 02:12:13,221    INFO MainThread azul.docker: Pulled image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467')
docker: invalid spec: :/home/developer/azul: empty section between colons

Run 'docker run --help' for more information
make: *** [Makefile:241: format] Error 125
make: Entering directory '/home/runner/work/azul/azul/lambdas'
for d in indexer service; do make -C $d openapi || ! break; done

```

The [last available build](https://github.com/DataBiosphere/azul/actions/runs/21229525197/job/61084770901#step:5:648) (085d7ec), prior to introducing #7777, succeeded in running `make format`:
```
2026-01-21 23:28:49,339    INFO MainThread __main__: No change for 'azul_terraform_version', still set to '1.13.5'
docker run \
    --attach stdout \
    --rm \
    --volume $(python scripts/resolve_container_path.py /home/runner/work/azul/azul):/home/developer/azul \
    --workdir /home/developer/azul \
    $(AZUL_DEBUG=0 python -m azul 'docker.resolve_docker_image_for_launch("pycharm")') \
    /opt/pycharm/bin/format.sh -r -settings .pycharm.style.xml -mask '*.py' src scripts test lambdas/layer/app.py lambdas/indexer/app.py lambdas/service/app.py .flake8/azul_flake8.py environment.py deployments/anvilbox/environment.py deployments/anvildev.browser/environment.py deployments/anvildev.gitlab/environment.py deployments/anvildev.shared/environment.py deployments/anvildev/environment.py deployments/anvilprod.browser/environment.py deployments/anvilprod.gitlab/environment.py deployments/anvilprod.shared/environment.py deployments/anvilprod/environment.py deployments/dev.browser/environment.py deployments/dev.gitlab/environment.py deployments/dev.shared/environment.py deployments/dev/environment.py deployments/hammerbox/environment.py deployments/prod.browser/environment.py deployments/prod.gitlab/environment.py deployments/prod.shared/environment.py deployments/prod/environment.py deployments/sandbox/environment.py deployments/tempdev.browser/environment.py deployments/tempdev.gitlab/environment.py deployments/tempdev.shared/environment.py deployments/tempdev/environment.py .github/workflows/schedule.py terraform/lambda_layer.tf.json.template.py terraform/cloudwatch.tf.json.template.py terraform/data_sources.tf.json.template.py terraform/backend.tf.json.template.py terraform/sqs.tf.json.template.py terraform/elasticsearch.tf.json.template.py terraform/step_function.tf.json.template.py terraform/common.tf.json.template.py terraform/dynamo.tf.json.template.py terraform/api_gateway.tf.json.template.py terraform/authentication.tf.json.template.py terraform/s3.tf.json.template.py terraform/providers.tf.json.template.py terraform/gitlab/gitlab.tf.json.template.py terraform/shared/shared.tf.json.template.py terraform/browser/browser.tf.json.template.py lambdas/indexer/.chalice/config.json.template.py lambdas/service/.chalice/config.json.template.py .github/pull_request_template.md.template.py
2026-01-21 23:28:50,075    INFO MainThread azul.docker: Resolving 'pycharm' image TagImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), tag='2025.2.6-71') …
2026-01-21 23:28:50,076    INFO MainThread azul.docker: Resolved 'pycharm' image to DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467')
2026-01-21 23:28:50,076    INFO MainThread azul.docker: Pulling image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467') …
2026-01-21 23:29:17,856    INFO MainThread azul.docker: Pulled image DigestImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), digest='sha256:dc096f0a93b4c20bb801888b27844986ffc24b362175ad0a7220d6c7319ce467')
Warning: [0.007s][warning][cds] Archived non-system classes are disabled because the java.system.class.loader property is specified (value = "com.intellij.util.lang.PathClassLoader"). To use archived non-system classes, this property must not be set
PyCharm 2025.2.6, build PC-252.28539.27 Formatter
Formatting /home/developer/azul/src/azul/exceptions.py...OK
Formatting /home/developer/azul/src/azul/azulclient.py...OK
…
Formatting /home/developer/azul/terraform/browser/browser.tf.json.template.py...OK
Formatting /home/developer/azul/lambdas/indexer/.chalice/config.json.template.py...OK
Formatting /home/developer/azul/lambdas/service/.chalice/config.json.template.py...OK
Formatting /home/developer/azul/.github/pull_request_template.md.template.py...OK

259 file(s) scanned.
259 file(s) formatted.
make: Entering directory '/home/runner/work/azul/azul/lambdas'
for d in indexer service; do make -C $d openapi || ! break; done
make[1]: Entering directory '/home/runner/work/azul/azul/lambdas/indexer'

```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -63,7 +63,7 @@ jobs:
           # an /etc/passwd that maps that to `developer`. We also need write
           # permissions for the group
           #
-          chmod -R g+w . && sudo chgrp -R 1000 . && make format && sudo chgrp -R $(id -g) .
+          chmod -R g+w . ; sudo chgrp -R 1000 . ; make format ; sudo chgrp -R $(id -g) .
           make -C lambdas openapi
           make -C .github
           make anvil_schema

